src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/domain/entities/AsylumCase.java

```diff
@@ -36,4 +36,7 @@ public AsylumCase() {
         this.put(asylumCaseDefinition.value(), value);
     }
 
+    public void clear(AsylumCaseDefinition asylumCaseDefinition) {
```

Comment:
why do you need this method?

Reply:
I need to clear the PAYMENT_FAILED_FOR_DISPLAY field over here: 

https://github.com/hmcts/ia-case-payments-api/pull/41/commits/98de05fd43f90d4b4e0f597b69574560191c9d81#diff-7450e6f7cdd076a3a63309d169676bc5R210

We have a similar asylumCase.clear method in ia-case-api (used throughout) so looked to introduce it into ia-case-payments-api